server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -431,7 +432,8 @@ final void importEvent(EventEnvelope event) {
             I id = ids.stream()
                       .findFirst()
                       .orElseThrow(() -> newIllegalStateException(
-                              "Unable to route import event `%s`.", messageType)
+                              "Unable to route import event `%s`. Event: %s",
+                              messageType, shortDebugString(message))
```

Comment:
Logging an event body is a security concern. Please only log the meta-data, such as event ID and timestamp.